Library/ViewModels/ChangePasswordViewModel.swift

```diff
@@ -48,30 +48,22 @@ ChangePasswordViewModelInputs, ChangePasswordViewModelOutputs {
       .merge(self.currentPasswordProperty.signal,
              self.onePasswordPrefillPasswordProperty.signal.skipNil())
 
-    let fieldsNotEmpty = Signal
-      .combineLatest(
-        currentPasswordSignal,
-        self.newPasswordProperty.signal,
-        self.confirmNewPasswordProperty.signal)
-      .map { valuesInFields in
-        return !valuesInFields.0.isEmpty && !valuesInFields.1.isEmpty && !valuesInFields.2.isEmpty
-    }
+    let fieldsNotEmpty = Signal.combineLatest(
```

Comment:
Common logic has been extracted in order for it to being shared between `CreatePasswordVM` & `ChangePasswordVM`